packages/patternfly-4/react-core/src/components/Form/ActionGroup.js

```diff
@@ -17,7 +17,7 @@ const defaultProps = {
 };
 
 const ActionGroup = ({ className, children, ...props }) => {
-  const customClassName = css(styles.formGroup, getModifier(styles, 'action', styles.modifiers.info), className);
+  const customClassName = css(styles.formGroup, getModifier(styles, 'action'), className);
```

Comment:
I missed this in the last review. it would probably be better to just use styles.modifiers.action
```suggestion
  const customClassName = css(styles.formGroup, styles.modifiers.action, className);
```

This would allow the babel plugin to throw at build time if this is ever modified. getModifier should only be used if the modifier is coming in via props.